src/code/fv-components/stable-drake.js

```diff
@@ -0,0 +1,24 @@
+import React, {PropTypes} from 'react';
+import OrganismView from '../components/organism';
+
+const StableDrakeView = ({org, id, width, height, style, onClick}) => {
```

Comment:
I don't quite understand the need for this view. It's just a single div wrapper around `OrganismView`, right? Couldn't this just be handled in the stable view? i.e. just by adding the wrapper where you currently have `stableDrakeViews = orgs.map...` in `fv-stable.js`?

I realize we already have a few views that just seem to be wrappers (and maybe those can be cleaned up too), but this in particular looks like it just has the one use-case.